cfgov/jinja2/v1/_layouts/spanish-ask-base.html

```diff
@@ -83,7 +83,7 @@
         </div><!-- .brand -->
 
         <nav role="navigation" class="main-nav js-showtoggle">
-            <button class="btn js-showtoggle-trigger s-show-on-small"><img src="{{ static('nemo/_/img/icon-menu-retina.png') }}" alt="Main Menu"></button>
+            <button class="a-btn js-showtoggle-trigger s-show-on-small"><img src="{{ static('nemo/_/img/icon-menu-retina.png') }}" alt="Main Menu"></button>
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/3092/commits/f13808069fbcf7ded59ba886d5170effbef81280